README.md

```diff
@@ -256,6 +256,23 @@ Since that process for running management commands is a pain, we recommend
 
 [wrapper-tool]: https://github.com/zulip/docker-zulip/wiki/Running-Management-Commands
 
+### Using a custom certificate bundle
+
+If you are sitting behind a custom CA and want to build the Zulip
+image yourself special care is required.
+
+Zulip installs packages via `yarn` and `pip` and these need to be
+configured to use these certificates. You will need to get your
+certificate bundle into the docker image, either by adding a `COPY`
+somewhere or by replacing the `FROM`s with your custom ubuntu image
+that includes your bundle. The recommended way is to have your own
+base image which has your bundle ready at the default
+`/etc/ssl/certs/ca-certificates.crt`.
+
+The next and last step is to set up the `CUSTOM_CA_CERTIFICATES`
+argument in `docker-compose.yml` to point to your CA bundle.
```

Comment:
I know this feels obvious, but we should explain what value to put there, i.e. the path to the bundle.